packages/devextreme/js/__internal/ui/chat/chat_message_list.ts

```diff
@@ -69,12 +76,18 @@ class MessageList extends Widget<MessageListOptions> {
     this._messageGroups?.push(messageGroup);
   }
 
+  _renderScrollable(): void {
+    this._scrollable = this._createComponent('<div>', Scrollable, {});
```

Comment:
Why we don't declare options: useKeyboard, showScrollbar mode, direction option?

Reply:
https://github.com/DevExpress/DevExtreme/pull/27863#discussion_r1701537214